apps/settings/elements/accessibility_audio.html

```diff
@@ -0,0 +1,19 @@
+<element name="accessibility-audio" extends="section">
+  <template>
+
+    <gaia-header action="back" data-href="#accessibility">
+      <h1 data-l10n-id="accessibility-audio"></h1>
+    </gaia-header>
+
+    <div>
+      <ul>
+        <li>
+          <gaia-switch id="monoaudio-enable" name ="accessibility.monoaudio.enable">
```

Comment:
Sorry to nit-pick, there's still one extra space after 'name' before '=', make sure it looks like ```name="accessibility.monoaudio.enable"```